### PR TITLE
Batu solutions

### PR DESCRIPTION
**Pull request by batuSRQ**

### CODE REVIEW

File: src/Week02/Batus_Solutions.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        System.out.println("Enter number to proceed : ");
+        int number = scanner.nextInt();
+        String result = "";
+
+
+        for (int i = 1; i < number; i++) {
+
+            if (i % 2 == 0 || i % 3 == 0 || i % 5 == 0) {
+                result = "";
+            }
```
Comment (dbmsch) [inline src/Week02/Batus_Solutions.java:27]: Hi! Everything looks good! But it seems that this statement needs to be removed according to task conditions.

File: src/Week02/Batus_Solutions.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+
+    }
+        public static void swap(int x,int y){
+            System.out.println("Before swap : " + x + " " + y);
+
+            x = x + y ;
+            y = x - y;
+            x = x - y;
```
Comment (dbmsch) [inline src/Week02/Batus_Solutions.java:59]: This one great! My code has one more step...

Comment (batuSRQ) [reply src/Week02/Batus_Solutions.java:59]: I checked your solution just before,It's almost same, actually I had no idea before I start to code that, i asked to my friend who does not know anything coding but he is good at Math,I told me way lol

File: src/Week02/Batus_Solutions.java (hunk 1 above)
Comment (batuSRQ) [reply src/Week02/Batus_Solutions.java:27]: That's right,I just realized that this code would run with same functionality without this statement.Thanks for heads up

**Pull request merged**